llvm/test/CodeGen/PowerPC/atomic-compare-exchange-weak.ll

```diff
@@ -0,0 +1,100 @@
+; Test the generate asm of function 
+; int foo(_Atomic int *cp, int *old, int c) {
+;  return atomic_compare_exchange_weak_explicit(cp, old, c, __ATOMIC_RELAXED, __ATOMIC_RELAXED);
+; }  
+
+; RUN: llc < %s -ppc-asm-full-reg-names  -mtriple=powerpc-ibm-aix7.2.0.0 -mcpu=pwr8  -verify-machineinstrs | FileCheck %s --check-prefix=CHECK
```

Comment:
Can `powerpc-ibm-aix-xcoff`/`powerpc64-ibm-aix-xcoff` be used instead, or do we have to use the 7.2 version of the triple?